masterlist.yaml

```diff
@@ -12970,6 +12975,16 @@ plugins:
         crc: 0x50EEF62E
         util: '[TES4Edit v4.0.3](https://www.nexusmods.com/oblivion/mods/11536)'
         itm: 138
+    clean:
+      - crc: 0x489D7BE6
+        util: 'TES4Edit v4.0.4c'
```

Comment:
```suggestion
```

We don't include cleaning data for dirty mods after they're cleaned since it's redundant (users will know if they cleaned the mod themselves) & volatile (different xEdit versions & QAC settings can produce different checksums).